site/e2e/tests/manageUsers.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { test, expect } from "@playwright/test";
+import { beforeCoderTest } from "../hooks";
+
+test.beforeEach(async ({ page }) => await beforeCoderTest(page));
+
+test("manage users", async ({ page, baseURL }) => {
+  await page.goto(`${baseURL}/users`, { waitUntil: "domcontentloaded" });
+  await expect(page).toHaveTitle("Users - Coder");
+
+  await page.getByRole("button", { name: "Create user" }).click();
+  await expect(page).toHaveTitle("Create User - Coder");
+
+  const userValues = {
+    username: "testuser",
+    email: "testuser@coder.com",
+    loginType: "password",
+    password: "s3cure&password!",
```

Comment:
This will be a problem if there are leftovers from previous test runs. Let's make them random, see `randomName()` in `helpers.ts`.

Reply:
If we want that, we should use a library IMO https://github.com/faker-js/faker

Reply:
I like it!